usmqe_tests/conftest.py

```diff
@@ -267,3 +267,29 @@ def fill_cpu():
             raise OSError(stderr)
         return request.param
     return measure_operation(fill_cpu)
+
+
+@pytest.fixture(params=[60, 80])
+def workload_memory_utilization(request):
+    """
+    Returns:
+        dict: contains information about `start` and `stop` time of stress-ng
+            command and its `result`
+    """
+    def fill_memory():
+        """
+        Use `stress-ng` tool to stress memory for 4 minutes to given percentage
+        """
+        # stress memory for for 240 seconds
+        run_time = 240
+        SSH = usmqe.usmssh.get_ssh()
+        host = pytest.config.getini("usm_cluster_member")
+        stress_cmd = "stress-ng --vm-method flip --vm {} --vm-bytes {}% --timeout {}s".format(
```

Comment:
To kee the results more stable, I would:

 * use `--vm-hang 0 --vm-keep --verify --syslog`